modules/sbo-unbinding-workloads-from-a-backing-service.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * /applications/connecting_applications_to_services/binding-workloads-using-sbo.adoc
+
+[id="sbo-unbinding-workloads-from-a-backing-service_{context}"]
+= Unbinding workloads from a backing service
+
+You can unbind a workload from a backing service by using the `oc` tool.
+
+. To unbind a workload from a backing service, delete the `ServiceBinding` custom resource (CR) linked to it :
+
+[source,terminal]
+----
+$ oc delete ServiceBinding <.metadata.name>
+----
+
+.Example
+[source,yaml]
+----
+$ oc delete ServiceBinding spring-petclinic-rest
+----
+where:
+[horizontal]
+`spring-petclinic-rest`:: Specifies the name of the `ServiceBinding` CR.
```

Comment:
```suggestion
```

Reply:
```suggestion
where `spring-petclinic-rest` specifies the name of the `ServiceBinding` CR.
```
@Srivaralakshmi The formatting of this bit in the preview doesn't look right with the capital letter `Specifies`. Is there a reason why you can't just put the "where" sentence all in one line as I suggest here?